src/sentry/static/sentry/app/components/sidebar/sidebarMenuItem.tsx

```diff
@@ -4,23 +4,22 @@ import {css} from '@emotion/core';
 
 import {Theme} from 'app/utils/theme';
 
-import SidebarMenuItemLink, {SidebarMenuItemLinkProps} from './sidebarMenuItemLink';
+import SidebarMenuItemLink from './sidebarMenuItemLink';
 import {OrgSummary} from './sidebarOrgSummary';
 
 type Props = {
   children: React.ReactNode;
-} & SidebarMenuItemLinkProps;
+} & React.ComponentProps<typeof SidebarMenuItemLink>;
 
-const SidebarMenuItem = ({to, href, children, ...props}: Props) => {
-  const hasMenu = !to && !href;
-  return (
-    <StyledSidebarMenuItemLink to={to} href={href} {...props}>
-      <MenuItemLabel hasMenu={hasMenu}>{children}</MenuItemLabel>
```

Comment:
We need to restore this because of https://github.com/getsentry/sentry/pull/17346/files#diff-4734084577830b057fc992bcdc606d26R44